src/webRTCEndpoint.ts

```diff
@@ -676,6 +676,14 @@ export class WebRTCEndpoint extends (EventEmitter as new () => TypedEmitter<
     simulcastConfig: SimulcastConfig = { enabled: false, activeEncodings: [] },
     maxBandwidth: TrackBandwidthLimit = 0 // unlimited bandwidth
   ): string {
+
+    const isUsedTrack = this.connection?.getSenders().some(val => val.track == track)
```

Comment:
```suggestion
    const isUsedTrack = this.connection?.getSenders().some(val => val.track === track)
```
Always use the triple-equals operator